content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -24,9 +24,7 @@ const CALENDAR_HEADINGS = {
 
 const ActivityFeedItem = React.createClass({
   getInitialState() {
-    return {
-      showContextMenu: false
-    };
+    return {showContextMenu: false};
```

Comment:
Was this a rule change? Or some `--fix` aggressive auto-format change?
Not sure moving to flattened objects is any easier to read (see line 105 below also).